results/tests/MetricRecorderTest.m

```diff
@@ -0,0 +1,68 @@
+classdef MetricRecorderTest < matlab.unittest.TestCase
+    
+    properties 
+        Config;
+        MetricRecorder;
+        Stations;
+        Users;
+    end
+
+    methods (TestClassSetup)
+        function createObjects(testCase)
+            testCase.Config = MonsterConfig();
+            %Set Harq active:
+            testCase.Config.Harq.active = true;
+            testCase.MetricRecorder = setupResults(testCase.Config);
+            testCase.Config.setupNetworkLayout();
+            testCase.Stations = setupStations(testCase.Config);
+            testCase.Users = setupUsers(testCase.Config);
+        end
+    end
+
+    methods (TestMethodTeardown)
+        function resetObjects(testCase)
+            %Test Teardown
+
+        end
+    end
+
+    methods (Test)
+        function testConstructor(testCase)
+            testCase.verifyTrue(isa(testCase.MetricRecorder,'MetricRecorder'));
+            testCase.verifyTrue(isa(testCase.MetricRecorder.Config,'MonsterConfig'));
+            testCase.verifyTrue(testCase.Config == testCase.MetricRecorder.Config);
+        end
+
+        function testRecordEnbMetrics(testCase)
+            testCase.MetricRecorder.recordEnbMetrics(testCase.Stations, testCase.Config);
+            %TODO: check for range and for all stations
+            testCase.verifyTrue(~isnan(testCase.MetricRecorder.util(1))); 
```

Comment:
Is it possible to test for known values? E.g. the metric recorder should copy relevant values from the eNB

Reply:
it should be possible. As the values are to be verified in the basic scenario test and simulation test I guess it would be redundant to also test it in this instance?